build/networking/README_DOCKER.md

```diff
@@ -130,7 +130,7 @@ configuration file settings and inner workings!
     mkdir -p "${VOLUME}"/intf
```

Comment:
The commands should be display text, not bullets:

   ```bash
   rm -rf ${VOLUME}/logs
   rm -rf ${VOLUME}/intf
   etc.
   ```

(The way you have them coded, the text is weirdly formatted when rendered by VS Code.)

I think you're better off not adding spaces around the less-than and greater-than characters. With the spaces, these look like shell redirect commands (`< Image` and `> rundaemon`). Better to write the metasymbol as `<ImageId>` (or maybe `${IMAGE_ID}`, since you use that elsewhere for replaceable text).

Yes, I know lint complains, but I think that's better than a misleading example in the documentation. Unfortunately, I can't find a way to escape them. Neither `\<` nor `&lt;` works, and I can't think of anything else to try.

Reply:
OK. I didnt see the markdown displayed properly - both VSCode and also the webpage just completely omits showing that text between `<` and `>` symbols.
Changing this to use `${IMAGE_ID}` instead.